wercker.yml

```diff
@@ -1,4 +1,12 @@
 box: node
+
+build:
+  steps:
+    - script:
+        name: Build and Test
+        code: |
+          echo "Look at me, building and testing..."
+
```

Comment:
This needs to run the gradle command to test, but I don't know that so that's something to research.